app/screens/FAQ.js

```diff
@@ -27,24 +27,32 @@ const FAQ = ({ navigation, }) => {
                         <Text style={s.text}>{`When you're on the map screen, click the "filter" button in the upper right, then choose a machine. Then go back to the map and it will only show places with that machine.`}</Text>
                         <Text style={s.bold}>{`This location closed/no longer has machines. What do I do - do I need to tell you?`}</Text>
                         <Text style={s.text}>{`Simply remove all the machines from it. Empty locations are periodically removed.`}</Text>
+                        <Text style={s.bold}>{`How do I get listed as an operator?`}</Text>
```

Comment:
This is totally fine, but with no variables or `""`, it could just be
```suggestion
                        <Text style={s.bold}>How do I get listed as an operator?</Text>
```